tests/test_project/tests/test_password_history.py

```diff
@@ -100,3 +100,230 @@ def test_change_number_hasher_iterations(self):
                     user_config__iterations=HistoryVeryStrongHasher.iterations).count(),
                 1,
             )
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
+        'NAME': 'django_password_validators.password_history.password_validation.UniquePasswordsValidator',
+        'OPTIONS': {
+            'lookup_range': 3
+        }
+    }])
+    def test_lookup_range(self):
+        """Tests considering the last 3 passwords and setting a valid password."""
+        self.create_user(1)
+        for i in range(4):
+            self.user_change_password(user_number=1, password_number=i)
+        for i in range(4):
+            self.assert_password_validation_False(
+                user_number=1, password_number=i)
+        self.assert_password_validation_True(user_number=1, password_number=4)
+
+    def test_lookup_range_string(self):
+        """Tests that a TypeError is raised if lookup_range is not an integer."""
+        with self.assertRaises(TypeError) as context:
+            validator = UniquePasswordsValidator(lookup_range='something')
+
+    def test_lookup_range_dictionary(self):
+        """Tests that a TypeError is raised if lookup_range is not an integer."""
+        with self.assertRaises(TypeError) as context:
+            validator = UniquePasswordsValidator(lookup_range={'foo': 1})
+
+    def test_lookup_range_list(self):
+        """Tests that a TypeError is raised if lookup_range is not an integer."""
+        with self.assertRaises(TypeError) as context:
+            validator = UniquePasswordsValidator(lookup_range=[2])
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
+        'NAME': 'django_password_validators.password_history.password_validation.UniquePasswordsValidator',
+        'OPTIONS': {
+            'lookup_range': 2
+        }
+    }])
+    def test_lookup_range_change_then_retry(self):
+        """Tests considering the last 2 passwords, entering a valid third one
+        then confirming it's no longer valid
+        """
+        self.create_user(1)
+        for i in range(3):
+            self.user_change_password(user_number=1, password_number=i)
```

Comment:
@spokerman12, since this code is repeated in multiple tests, can you refactor it into a common utility method that can be used everywhere? Something like

```python
self.user_change_password_number_of_times(3)
```
?